java/src/main/kotlin/io/spine/validation/java/PatternFieldGenerator.kt

```diff
@@ -99,9 +99,9 @@ internal class PatternFieldGenerator(private val view: PatternField) {
         }
 
         else -> error {
-            "Field type `${fieldType.name}` is not supported by `PatternFieldGenerator`. " +
-                    "Please ensure that the supported field types in this generator match those " +
-                    "used by `PatternPolicy` when validating the `PatternFieldDiscovered` event."
+            "Field type `${fieldType.name}` is not supported by `PatternFieldGenerator`." +
```

Comment:
Same comments here. Isn't it `The field type`? And, isn't it a compilation error?

Reply:
> Is it possible not to get into this branch at all, instead of explaining the error at this level of details?

It'd be great. But I haven't figured out how to make it. 

The policy checks the field type, it can bypass the resolved type to the view. I've tried to do this with per-policy enums, but continued running into non-exhaustive `when` because Protobuf enums inevitably have some service constants.